Makefile

```diff
@@ -48,6 +59,142 @@ integration: build-skyconf ## runs integration tests.
 build-skyconf: ## builds skyconf.
 	${OPTS} go install ./cmd/skyconf
 
+skybian-skywire-package-amd64: #native build
```

Comment:
I feel like we can put the majority of this logic here into scripts. This way, we wouldn't need so much code duplication.

Reply:
yes